### ISSUE TITLE
View Editor: copy/cut/paste issues

### ISSUE THREAD

**Issue by opoupeney** [InteractiveClouds/dfx#2]
There are several issues in the copy/cut/paste in the view editor:

- Paste works only when a container type of GC (panel / tab / wizard) is selected, it should take the parent container if a non-container GC is selected
- Duplicate and Copy have the same icon. Duplicate must be illustrated with a "+" sign
- Paste works only once (cannot paste twice a same copied GC)
- Copy/Cut/Paste breaks the undo button (seems the history is cleared)

**Comment by vinnie777** [comment]
Made serious refactoring and fixed:

1) These issues:
* Paste works only when a container type of GC (panel / tab / wizard) is selected, it should take the parent container if a non-container GC is selected
* Duplicate and Copy have the same icon. Duplicate must be illustrated with a "+" sign
* Paste works only once (cannot paste twice a same copied GC)
2) Did many tests, found and fixed these issues as well:
* Copy panel containing components to another card
* Copy tab to another tab, or tab to wizard
* Copy wizard to another wizard or to tab
* Copy panel containing components to another panel of top level
* Copy panel to another panel of second level
* Copy tab or wizard to another card
